src/mon/MgrMonitor.cc

```diff
@@ -1271,10 +1282,13 @@ bool MgrMonitor::prepare_command(MonOpRequestRef op)
       r = -EINVAL;
       goto out;
     }
-    if (pending_map.get_always_on_modules().count(module) > 0) {
+
+    if (pending_map.force_disabled_modules.count(module) == 0 &&
```

Comment:
nit: use `contains`.

Reply:
... and in other occurrences wherever possible.

Reply:
Done.